package.json

```diff
@@ -48,8 +48,9 @@
     "@nestjs/schedule": "^2.1.0",
     "@nestjs/swagger": "^6.2.1",
     "@nestjs/typeorm": "^9.0.1",
+    "@nestlab/google-recaptcha": "^3.2.0",
     "@sindresorhus/slugify": "^1.1.2",
-    "axios": "^1.3.2",
+    "axios": "^1.2.1",
```

Comment:
Why the downgrade? https://github.com/chvarkov/google-recaptcha/blob/master/package.json#L38 supports anything between v1 and v2 of axios.

Reply:
Oh, saw your comment on that in the description.

Reply:
https://github.com/chvarkov/google-recaptcha/releases/tag/v3.2.0 should now support latest Nest 9. I don't see the discrepancy with the versions.

Reply:
It's not the Nest version itself, it's the peer dependencies that won't work. The recaptcha module supports versions of axios from v1 to v2, v1 being inlusive and v2 being exclusive, i.e. it doesn't support v2.

Reply:
oh, silly me, just noticed the change on the other line for `@nestjs/axios`. Got it now.